front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/CraftComponents/HomepageBanner/index.tsx

```diff
@@ -255,28 +266,28 @@ const HomepageBannerSettings = () => {
       }
     });
     if (value !== 'customized_button') {
+      const newErrorTypes = errors?.filter(
```

Comment:
I've moved these error states away from the props and into a component state in order to ensure they are not stored in the craftjs json.